packages/genesys-spark-components/src/i18n/translations/en.json

```diff
@@ -99,6 +99,9 @@
     "increment": "Increment",
     "decrement": "Decrement"
   },
+  "gux-form-field-optional-indicator": {
+    "optional": "optional"
```

Comment:
Does optional need to uppercase or is lowercase expected?

Reply:
good question, it's lowercase by design